packages/sdk/package.json

```diff
@@ -59,4 +63,4 @@
   "overrides": {
     "viem": "2.27.0"
   }
-}
```

Comment:
please add this line back :)

Reply:
I think this is a bug in GH - I believe my autoformatter removed one-too-many empty lines. `}` is still there and `package.json` is still a valid JSON.